helm/deletion-blocker-operator/values.schema.json

```diff
@@ -149,6 +152,12 @@
         "securityContext": {
             "type": "object",
             "properties": {
+                "allowPrivilegeEscalation": {
+                    "type": "boolean"
+                },
+                "capabilities": {
```

Comment:
We use toYaml in templates so it doesn't make sense to add whole pod schema here.